hal/ubuntu/README.md

```
@@ -1,11 +1,154 @@
+The custom kernel must be built with the following options:
+- `CONFIG_AMD_MEM_ENCRYPT=y` — AMD memory encryption support
+- `CONFIG_SEV_GUEST=y` — SEV guest driver
+- Coconut SVSM guest support patches applied
+
+The kernel must be packaged as `.deb` files.
+
+**Installing the kernel into the seed image:**
+
+Place the `.deb` files in a `debs/` directory next to `qemu.sh`:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)